.buildkite/scripts/steps/functional/security_solution.sh

```diff
@@ -6,6 +6,8 @@ source .buildkite/scripts/steps/functional/common.sh
 
 export JOB=kibana-security-solution-chrome
 export KIBANA_INSTALL_DIR=${KIBANA_BUILD_LOCATION}
+# export BUILDKITE_ANALYTICS_TOKEN="$(retry 5 5 vault read -field=security_solution secret/siem-team/kibana/buildkite_test_analytics_cypress)"
```

Comment:
Typically at `.buildkite/scripts/lifecycle/pre_command.sh`.  If this token is SIEM specific it may be worth prefixing the variable as a signal to other teams.

Reply:
it seems I don't have access to read/write to `secret/kibana-issues/dev` should I open some issue to grant me access or something else?